### PR TITLE
New example code for facebook research segment anything

### PR DESCRIPTION
**Pull request by jleibs**
Facebook Research put out a new segmentation model today that looked cool, so I wanted to play around with it.

https://segment-anything.com/

This is just a minimal demo that downloads the model, and uses the `SamAutomaticMaskGenerator` on a collection of images. Lots of other cool functionality that could be added in the future when it comes to interactivity.

By default it runs on the sample image:
```
pip install -r examples/python/segment_anything/requirements.txt 
examples/python/segment_anything/main.py
```
image

But you can also tell it to run using cuda, or pass other images such as:
```
python main.py --device cuda https://raw.githubusercontent.com/facebookresearch/segment-anything/main/notebooks/images/dog.jpg
```

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)

### CODE REVIEW

File: examples/python/segment_anything/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+numpy
+torch
+torchvision
+opencv-python
+requests
+rerun-sdk
+tqdm
+-e git+https://github.com/facebookresearch/segment-anything.git#egg=segment-anything
```
Comment (teh-cmc) [inline examples/python/segment_anything/requirements.txt:8]: nit: sort

**Review by teh-cmc — APPROVED** [review]
That's so cool, and it integrates so smoothly with Rerun :two_hearts:

**Comment by emilk** [comment]
Very cool indeed, but would it be possible to add labels?

image

**Comment by jleibs** [comment]
> Very cool indeed, but would it be possible to add labels?

There’s not really anything to label it with since it’s not categorizing anything — just segmenting them. I can re-use class-id as label so the field is at least populated though.

**Pull request merged**